src/components/SlateEditor/plugins/mathml/EditMath.js

```diff
@@ -32,13 +32,11 @@ class EditMath extends Component {
   }
 
   componentDidMount() {
+    // force set state to trigger rerender.
+    this.setState({});

```

Comment:
Hvorfor må denne være her? Vil den ikke fjerne det som blir satt i constructor?

Reply:
Nei, den vil merges med det som settes i konstruktøren. 

> You may optionally pass an object as the first argument to setState() instead of a function. This performs a shallow merge of stateChange into the new state. https://reactjs.org/docs/react-component.html#setstate

Og fra https://reactjs.org/docs/react-component.html#componentdidmount

> You may call setState() immediately in componentDidMount(). It will trigger an extra rendering, but it will happen before the browser updates the screen. This guarantees that even though the render() will be called twice in this case, the user won’t see the intermediate state.

Uten denne vil forhåndsvisninga i mattedialogen være uformatert ved første visning. Alternativet var å tvinge en oppdatering via this.forceUpdate(), men eg syntes dette virka sikrere.